client/src/tokens/colors.js

```diff
@@ -0,0 +1,266 @@
+/** @typedef {string} ColorHex */
+/** @typedef {string} ColorJS */
+/** @typedef {string} ColorCSS */
+
+/** @typedef {{
+    hex: ColorHex;
+    js: ColorJS;
+    css: ColorCSS;
+    bgUtility: ColorCSS;
+    textUtility: ColorCSS;
+    usage: string;
+    contrastText: keyof typeof allColors;
+}} Shade */
+
+/** @typedef {{
+    [jsName: string]: Shade;
+}} Color */
+
+/** @type {Color} */
+const colors = {
```

Comment:
Maybe it would be worthwhile to split these into separate files or `const` per color 🤔